### PR TITLE
Fix persisted app id on launch

### PR DESCRIPTION
**Pull request by nporter-adbe**
Addresses: https://github.com/adobe/aepsdk-core-ios/issues/459

### CODE REVIEW

File: AEPCore/Sources/configuration/Configuration.swift [hunk 1]
```diff
@@ -46,11 +46,6 @@ class Configuration: Extension {
         registerListener(type: EventType.configuration, source: EventSource.requestContent, listener: receiveConfigurationRequest(event:))
         registerListener(type: EventType.lifecycle, source: EventSource.responseContent, listener: receiveLifecycleResponse(event:))
 
-        // If we have an appId stored in persistence, kick off the configureWithAppId event
-        if let appId = appIdManager.loadAppId(), !appId.isEmpty {
```
Comment (shalehaha) [inline AEPCore/Sources/configuration/Configuration.swift:50]: `loadAppId ` also loads the id from the manifest, so we couldn't simply remove it. May need to directly use `loadAppIdFromPersistence() or  loadAppIdFromManifest()` here.

Comment (nporter-adbe) [reply AEPCore/Sources/configuration/Configuration.swift:50]: good point, fixed.

File: AEPCore/Tests/FunctionalTests/Configuration/ConfigurationLifecycleResponseTests.swift [hunk 2]
```diff
@@ -80,36 +80,4 @@ class ConfigurationLifecycleResponseTests: XCTestCase {
         mockSystemInfoService.property = nil
     }
 
-    /// Tests that app id is loaded from manifest on normal startup after being persisted
-    func testHandleValidAppidFromManifestPersisted() {
```
Comment (shalehaha) [inline AEPCore/Tests/FunctionalTests/Configuration/ConfigurationLifecycleResponseTests.swift:84]: should we keep this test?

Comment (nporter-adbe) [reply AEPCore/Tests/FunctionalTests/Configuration/ConfigurationLifecycleResponseTests.swift:84]: I don't believe so. The first part of the test ensures that the app is loaded from the manifest, this part is still valid. But the second half of the tests removes the app if from the manifest and checks if the app is is loaded from persistence which is no longer valid. 

Maybe we can keep the first half of the test? I'd assume we already have a test for that case.

**Review by shalehaha — APPROVED** [review]

**Pull request merged**